config/sidekiq_cron_schedule.yml

```diff
@@ -38,6 +38,10 @@ sync_hesa_students:
   cron: "0 5 * * *"
   class: "Hesa::SyncStudentsJob"
   queue: hesa
+sync_dqt_teachers:
+  cron: "0 0 1 * *"
```

Comment:
Run the job as soon as possible on the first day of each month. I'm assuming here that we don't want to hit the DQT API very often.

Reply:
question because i think i missed the background for this ticket - what are we going to be using these tables for? are we ok with them becoming a month out of date? when we were syncing trainee states from DQT (similar batch process) we were doing this daily (but were thinking about reducing it to weekly)

cc @claire-hughez

Reply:
thinking about it - the trainee scope in this PR looks to be much bigger though (before it was scoped to HESA trainees only) so many more calls to the API..

Reply:
I think we should do this is a one-off exercise for now from my perspective, which we may need to repeat at a later point manually. I personally don't think we need to get it to repeat at a certain amount of time yet, especially considering the trainee scope and number of requests it'll put on DQT.

Like John H said before though, we thought this with the hesa_students table and now we'll do it more, but we'll just see how it goes and then make any decisions on repeating.

Reply:
I'm removing cron. We just kick it off manually.